src/main/java/by/onliner/retry/RetryAnalyzer.java

```diff
@@ -0,0 +1,30 @@
+package by.onliner.retry;
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+/***
+ * RetryAnalyzer helps to check if the failure was caused by a network glitch, random machine issues, server issues
+ * like unexpected delay in the response from server.
+ */
+public class RetryAnalyzer implements IRetryAnalyzer {
+
+    private int actualRetry;
+    private static final int MAX_RETRY = 3;
```

Comment:
i think we should update this value so that the test is rerun max 2 times

Reply:
Fixed